lib/grid_printer.rb

```diff
@@ -0,0 +1,12 @@
+class GridPrinter
+  attr_reader :stdout, :grid
+
+  def initialize(stdout, grid)
+    @stdout = stdout
+    @grid = grid
+  end
+
+  def print_grid
+    stdout.puts(grid.current_grid)
```

Comment:
This class seems unnecessary as it is right now. The grid object is doing all the real work of associating data with the template, so that work should come live here.